### PR TITLE
Migate tabs to fela

### PR DESCRIPTION
**Pull request by koddsson**
I added a second commit that fixes this gap in the examples:

Before:

image

After:

image

### CODE REVIEW

File: packages/cf-component-tabs/src/Tabs.js [hunk 1]
```diff
@@ -13,6 +14,85 @@ const find = (list, condition) => {
   return foundElement;
 };
 
+const styles = ({ theme }) => ({
+  marginTop: '1.5rem',
+  marginBottom: '1.5rem',
+  border: '1px solid #dedede'
```
Comment (manatarms) [inline packages/cf-component-tabs/src/Tabs.js]: Theme color?

Comment (koddsson) [inline packages/cf-component-tabs/src/Tabs.js]: I couldn't find any `#dedede` color but I'll do a manual comparison and find a close one for this whole PR.

File: packages/cf-component-tabs/src/TabsPanel.js [hunk 2]
```diff
@@ -1,14 +1,24 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  backgroundColor: theme.backgroundColor
+  //borderColor: theme.borderColor,
```
Comment (manatarms) [inline packages/cf-component-tabs/src/TabsPanel.js]: Any reason these are commented out? Can we remove them if they're not used?

File: packages/cf-component-tabs/src/TabsPanelTheme.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export default baseTheme => ({
+  backgroundColor: baseTheme.colorWhite,
+  borderColor: '#dedede',
```
Comment (manatarms) [inline packages/cf-component-tabs/src/TabsPanelTheme.js]: Style const?

File: packages/cf-component-tabs/src/TabsTheme.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  },
+
+  '&:focus': {
+    '&::after': {
+      outline: 'none'
+    }
+  },
+
+  '&:hover': {
+    background: '#f6f6f6',
```
Comment (manatarms) [inline packages/cf-component-tabs/src/TabsTheme.js]: Style const?

File: packages/cf-component-tabs/src/TabsPanel.js (hunk 2 above)
Comment (koddsson) [inline packages/cf-component-tabs/src/TabsPanel.js]: Yup, I'll remove this

File: packages/cf-component-tabs/src/Tabs.js (hunk 1 above)
Comment (neynah) [inline packages/cf-component-tabs/src/Tabs.js]: Smoke (#e0e0e0) is quite close.

File: packages/cf-component-tabs/src/TabsTheme.js (hunk 4 above)
Comment (neynah) [inline packages/cf-component-tabs/src/TabsTheme.js]: Pretty close to Hail #F7F7F7

Comment (koddsson) [inline packages/cf-component-tabs/src/TabsTheme.js]: Looks good to me, I'll switch it out.

`#f6f6f6` vs. `#f7f7f7`

**Pull request merged**